### PR TITLE
Update TRANSLATION_GUIDE.md

### PR DESCRIPTION
**Pull request by uddk6215**
공식 용어집에 새로운 용어를 제안

## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #819 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [ ] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [x] 위 종류에 포함되지 않는 기여

## PR 설명
2가지의 용어가 추가됩니다.
- Gradient descent : 경사하강법
- Pooling : 풀링

### CODE REVIEW

File: TRANSLATION_GUIDE.md [hunk 1]
```diff
@@ -68,6 +68,7 @@
 |Generative|생성 모델|박정환|ToC의 분류명입니다.|
 |Getting Started tutorial|시작하기 튜토리얼|박정환|ToC의 Getting Started를 뜻합니다.|
 |gradient|변화도|박정환||
+|Gradient descent|경사하강법|이진혁|
```
Comment (rumjie) [inline TRANSLATION_GUIDE.md:71]: 여기 추가 설명하실 부분이 없으면 | 하나가 더 들어가야 테이블 컬럼 수와 맞을거 같습니다~!

Comment (uddk6215) [reply TRANSLATION_GUIDE.md]: 반영했습니다. :)
